boards/arm/stm32f769i_disco/stm32f769i_disco.dts

```diff
@@ -23,8 +23,10 @@
 	};
 
 	sdram1: sdram@c0000000 {
+		compatible = "mmio-sram";
 		device_type = "memory";
```

Comment:
Looking at this (and not related to this PR):
@mbolivar-nordic: device_type prop is nowhere documented in bindings. Shouldn't it be removed ?

Reply:
Curious as well, shows up in `edtlib.py` but I don't think it's used anywhere and it's declared as deprecated in the specs https://github.com/devicetree-org/devicetree-specification/blob/main/source/chapter2-devicetree-basics.rst#device_type-deprecated.

Reply:
> it's declared as deprecated in the specs https://github.com/devicetree-org/devicetree-specification/blob/main/source/chapter2-devicetree-basics.rst#device_type-deprecated.

@fabiobaltieri the link you pasted says (emphasis mine):

>  new use of the property is deprecated, and **it should be included only on cpu and memory nodes** for compatibility with IEEE 1275–derived devicetrees.

So from a specification point of view, it seems fine to keep this as is.

@galak asked about this on the spec mailing list, but the results were inconclusive: https://www.spinics.net/lists/devicetree-spec/msg00800.html

Take from that what you will.